src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -18,7 +18,8 @@
             "polyfills": "src/polyfills.ts",
             "assets": [
               "src/assets",
-              "src/favicon.ico"
+              "src/favicon.ico",
+	            "src/branding/assets"
```

Comment:
Can you set the appropriate indentation?

Reply:
@alfonsomthd , its aligned properly, you can check here https://github.com/rhcs-dashboard/ceph/blob/586aa7010bd0f29b3675a1c76e7cfce4ff9765cd/src/pybind/mgr/dashboard/frontend/angular.json, but somehow GitHub is showing it misaligned.